v2/metadata/client.go

```diff
@@ -126,12 +131,33 @@ type Pipeline struct {
 	pipelineRunCtx *pb.Context
 }
 
+func (p *Pipeline) GetRunCtxID() int64 {
+	if p == nil {
+		return 0
```

Comment:
Do we need to throw error here?

Reply:
I think `return 0` is equal to returning an error, there's not much need to add separate error return.